<launch>
  <arg name="eskf_lio_config_path" default="$(find-pkg-share eskf_lio)/config/eskf_lio.params.yaml"/>

  <node pkg="eskf_lio" exec="eskf_lio_node" name="eskf_lio" output="screen">
    <remap from="points_raw" to="/sensing/lidar/left_upper/pointcloud_raw"/>
    <remap from="imu_raw" to="/sensing/imu/tamagawa/imu_raw"/>
    <param from="$(var eskf_lio_config_path)"/>
  </node>

  <node pkg="rviz2" exec="rviz2" name="rviz2" args="-d $(find-pkg-share eskf_lio)/rviz/eskf_lio.rviz"/>
</launch>
